addons/website_sale/views/templates.xml

```diff
@@ -836,43 +842,22 @@
 
         <a
             t-att-href="keep('/shop/category/' + slug(c), category=0)"
-            class="text-decoration-none p-0 text-900"
-        >
-            <div class="form-check d-inline-block">
-                <input
-                    type="radio"
-                    t-attf-name="wsale_categories_radios_{{parentCategoryId}}"
-                    class="form-check-input pe-none "
-                    t-att-id="c.id"
-                    t-att-value="c.id"
-                    t-att-checked="'true' if c.id == category.id else None"/>
-                <label class="form-check-label fw-normal" t-field="c.name"/>
-            </div>
-        </a>
+            t-attf-class="{{c.id == category.id and 'text-decoration-underline'}} p-0"
+            t-field="c.name"
+        />
     </template>
 
     <template id="website_sale.products_categories_list" active="True" name="eCommerce Categories">
         <h6 t-attf-class="o_categories_collapse_title mb-3 {{_titleClasses}}"><b>Categories</b></h6>
 
-        <div class="wsale_products_categories_list">
-            <ul class="nav d-flex flex-column my-2">
+        <div t-attf-class="wsale_products_categories_list">
```

Comment:
This was made on purpose to avoid a specific issue described here : https://github.com/odoo/odoo/pull/177056#discussion_r1723283881

Perhaps you would have another solution to fix that ? 🙂

Reply:
Give the divs a `name` and xpath it instead of the classes?